app/gimlet/app.toml

```diff
@@ -145,10 +155,10 @@ path = "../../drv/gimlet-seq-server"
 name = "drv-gimlet-seq-server"
 features = ["h753"]
 priority = 3
-requires = {flash = 32768, ram = 1024 }
+requires = {flash = 65536, ram = 8192 }
```

Comment:
So, rodata in this change is twice as large as text. Are you aware that you appear to have pulled in unicode grapheme tables and a bunch of supporting string stuff? What sort of string manipulation is this change doing?

Reply:
So, the increase in rodata seems to fall into a few categories.

- There are tables for a lot of enums giving the Debug representation, which seem to be almost entirely `unwrap()` on IPC returns
- There's a large binary data table I can't really explain.
- Panic messages.